### ISSUE TITLE
options.json doesn't seem to be read

### ISSUE THREAD

**Issue by xChickens** [j-hc/revanced-magisk-module#376]
I'm getting the following even though options is configured as according to https://j-hc.github.io/rvmm-config-gen/.

```
app.revanced.patcher.patch.PatchException: The option client-id requires a value, but null was passed
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:185)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:203)
	at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt)
	at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230)
	at app.revanced.cli.command.PatchCommand$run$4$patcherResult$1$1.invokeSuspend(PatchCommand.kt:302)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
	at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:301)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2026)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12)
Caused by: app.revanced.patcher.patch.options.PatchOptionException$ValueRequiredException: The option client-id requires a value, but null was passed
	at app.revanced.patcher.patch.options.PatchOption.assertRequiredButNotNull(PatchOption.kt:76)
	at app.revanced.patcher.patch.options.PatchOption.getValue(PatchOption.kt:58)
	at app.revanced.patcher.patch.options.PatchOption.getValue(PatchOption.kt:88)
	at app.revanced.patches.reddit.customclients.AbstractSpoofClientPatch.getClientId(AbstractSpoofClientPatch.kt:29)
	at app.revanced.patches.reddit.customclients.syncforreddit.api.SpoofClientPatch.patchClientId(SpoofClientPatch.kt:38)
	at app.revanced.patches.reddit.customclients.AbstractSpoofClientPatch$execute$1.invoke(AbstractSpoofClientPatch.kt:46)
	at app.revanced.patches.reddit.customclients.AbstractSpoofClientPatch$execute$1.invoke(AbstractSpoofClientPatch.kt:46)
	at app.revanced.patches.reddit.customclients.AbstractSpoofClientPatch.execute$executePatch(AbstractSpoofClientPatch.kt:44)
	at app.revanced.patches.reddit.customclients.AbstractSpoofClientPatch.execute(AbstractSpoofClientPatch.kt:46)
	at app.revanced.patches.reddit.customclients.AbstractSpoofClientPatch.execute(AbstractSpoofClientPatch.kt:11)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:174)
	... 24 more
 ```

**Comment by xChickens** [comment]
Just added `--options=options.json` to https://github.com/j-hc/revanced-magisk-module/blob/144b773b2c281efbd9fc58d80349b3500a248821/utils.sh#L330 and looks like it works.

**Comment by j-hc** [comment]
yes you are right. this was probably changed with the last breaking changes to the cli. fixed now a0eb9e9
